### DISCUSSION TITLE
Enhance the management capabilities of the Paimon format

### DISCUSSION THREAD

**Discussion by zhoujinsong** [apache/amoro#2538] [Ideas]
Currently, Amoro provides only basic management capabilities for the Paimon format. We should attempt to further expand additional management capabilities to help Paimon users achieve a better user experience.

**Comment by xieyi888** [comment]
I had some ideas

- Amoro handle compact for paimon automatically. 
Use paimon dedicated compaction as optimize task, So we can use existing optimizer group to manage compact for paimon, avoid adding single dedicated compaction  job(https://paimon.apache.org/docs/master/maintenance/dedicated-compaction/#dedicated-compaction-job) , which will lead to  operation and maintenance inconvenience and also increase resource

- Amoro handle remove orphan files for paimon automatically,  to avoid deployiong single remove_orphan_files,job(https://paimon.apache.org/docs/master/maintenance/manage-snapshots/#remove-orphan-files) for paimon, which will lead to  operation and maintenance inconvenience and also increase resource
